WPFSKillTree/Utils/UrlProcessing/BanditConverter.cs

```diff
@@ -0,0 +1,56 @@
+using System;
+using POESKillTree.Model;
+
+namespace POESKillTree.Utils.UrlProcessing
+{
+    /// <summary>
+    /// Provides functionality for converting bandits identifiers from external sources into used in the POESKillTree application.
+    /// </summary>
+    public class BanditConverter
```

Comment:
Hm, it would look lot cleaner imo and I don't think someone would misinterpret it. But I wouldn't really mind if it stay as is.